sql/src/main/java/io/crate/metadata/cluster/RenameTableClusterStateExecutor.java

```diff
@@ -22,219 +22,31 @@
 
 package io.crate.metadata.cluster;
 
-import io.crate.Constants;
 import io.crate.execution.ddl.tables.RenameTableRequest;
-import io.crate.metadata.IndexParts;
-import io.crate.metadata.PartitionName;
 import io.crate.metadata.RelationName;
-import io.crate.metadata.doc.DocIndexMetaData;
-import org.apache.logging.log4j.Logger;
-import org.elasticsearch.ElasticsearchException;
-import org.elasticsearch.action.support.IndicesOptions;
 import org.elasticsearch.cluster.ClusterState;
-import org.elasticsearch.cluster.block.ClusterBlocks;
-import org.elasticsearch.cluster.metadata.AliasAction;
-import org.elasticsearch.cluster.metadata.AliasMetaData;
-import org.elasticsearch.cluster.metadata.IndexMetaData;
 import org.elasticsearch.cluster.metadata.IndexNameExpressionResolver;
-import org.elasticsearch.cluster.metadata.IndexTemplateMetaData;
-import org.elasticsearch.cluster.metadata.MetaData;
 import org.elasticsearch.cluster.metadata.MetaDataIndexAliasesService;
-import org.elasticsearch.common.logging.Loggers;
 import org.elasticsearch.common.settings.Settings;
-import org.elasticsearch.common.xcontent.DeprecationHandler;
 import org.elasticsearch.common.xcontent.NamedXContentRegistry;
-import org.elasticsearch.common.xcontent.XContentParser;
-import org.elasticsearch.common.xcontent.json.JsonXContent;
-import org.elasticsearch.index.Index;
-import org.elasticsearch.index.IndexNotFoundException;
-import org.elasticsearch.indices.IndexTemplateMissingException;
 
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-
-public class RenameTableClusterStateExecutor extends DDLClusterStateTaskExecutor<RenameTableRequest> {
-
-    private static final IndicesOptions STRICT_INDICES_OPTIONS = IndicesOptions.fromOptions(false, false, false, false);
-
-    private final Logger logger;
-    private final IndexNameExpressionResolver indexNameExpressionResolver;
-    private final MetaDataIndexAliasesService metaDataIndexAliasesService;
-    private final NamedXContentRegistry xContentRegistry;
-    private final DDLClusterStateService ddlClusterStateService;
+public class RenameTableClusterStateExecutor extends AbstractRenameTableClusterStateExecutor {
```

Comment:
I'd remove the Abstract one and just extend the Exchange class from this one. Currently this child inherit everything from the abstract...

Reply:
not relevant anymore